extensions/wikia/AdEngine/js/template/bfaaDesktop.js

```diff
@@ -113,6 +145,41 @@ define('ext.wikia.adEngine.template.bfaaDesktop', [
 		}
 
 		log(['show', params.uap], log.levels.info, logGroup);
+
+		// TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)